### ISSUE TITLE
Datasource does not support alerting queries

### ISSUE THREAD

**Issue by SerhiiAksiutin** [logzio/grafana-logzio-datasource#2]
Hi,

I am trying to set up an alert for Hipchat (Load test room) using Grafana + Logz io (Quota View panel)

On alert tab in Grafana I am getting "The datasource does not support alerting queries" and when I try to test the rule getting this error:
```
firing:true
state:"alerting"
conditionEvals:" = true"
timeMs:"3.983ms"
error:"tsdb.HandleRequest() error Could not find executor for data source type: logzio"\
```

<img width="1336" alt="screen shot 2018-08-30 at 4 08 31 pm" src="https://user-images.githubusercontent.com/27512156/44884157-18124d00-ac6f-11e8-851e-e61bbc682712.png">

The alert doesn't appear to work because of this. But if I pause it and then resume I will get desired notification in hipchat with the error (because of "If execution error or timeout":"set alerting state" rule)

<img width="1179" alt="screen shot 2018-08-30 at 4 13 23 pm" src="https://user-images.githubusercontent.com/27512156/44884541-bfdc4a80-ac70-11e8-9391-bd90e4aacdd5.png">

Source:
https://logz.io/blog/grafana-integration/
https://logz.io/blog/enhancing-account-utilization/

**Comment by eeddaann** [comment]
I encountered the same issue and found a kludge solution - not recommended but better than nothing.. 
**tl;dr:** It actually overrides elasticsearch's built-in datasource (which supports alerting) to send logzio's header and it works like a charm - so no need for this datasource.    
- Search in Grafana's sources the expression: ```t,method``` (I found it grafana-5.3.0/public/build/5.770fad3a626c70250d5f.js)
- Inject the token like ```t,headers : {"X-API-TOKEN" : "XXXX"},method```
- save the file and that's it.
**Some background:**
- The alerting engine in grafana is in the backend of grafana - that's what makes the alerting not flexible when it comes to plugins
- behind the scenes this solution "merge" [datasource.ts](https://github.com/logzio/grafana-logzio-datasource/blob/master/src/datasource.ts) in this repo to [datasource.ts](https://github.com/grafana/grafana/blob/master/public/app/plugins/datasource/elasticsearch/datasource.ts) in Grafana's repo

**Comment by SerhiiAksiutin** [comment]
Btw, I have contacted Logz.io support for the matter. They have created a future request without ETA. Future request ID is... Actually they didn't provide with one.
